lib/slack.rb

```diff
@@ -1,11 +1,103 @@
 #!/usr/bin/env ruby
+require 'httparty' 
+require 'awesome_print'
+require 'dotenv'
+require_relative '../lib/recipient'
+require_relative '../lib/channel'
+require_relative '../lib/user'
+Dotenv.load
+# require 'table_print'
+
+class SlackApiError < StandardError; end
+
 
 def main
-  puts "Welcome to the Ada Slack CLI!"
+  ap "Welcome to the Ada Slack CLI!"
+
+  def menu
+    puts
+    ap "Main Menu"
+    puts "a. list users"
+    puts "b. select user"
+    puts "c. list channels"
+    puts "d. select channel"
+    puts "e. details for selected user and channel"
+    puts "f. send message"
+    puts "g. quit\n\n"
+
+    ap "What would you like to do?"
```

Comment:
BTW thank you for letting me select a letter for an option rather than typing out the entire "list users" etc.